L1Trigger/L1THGCal/test/HGCalTriggerGeomTesterV9Imp2.cc

```diff
@@ -831,14 +980,20 @@ void HGCalTriggerGeomTesterV9Imp2::fillTriggerGeometry()
       ic++;
     }
 
-    treeTriggerCells_->Fill();
+    // TEMPORARY
+    //    treeTriggerCells_->Fill();
+
     // fill modules
     uint32_t module = triggerGeometry_->getModuleFromTriggerCell(id);
     auto itr_insert = modules.emplace(module, std::unordered_set<uint32_t>());
     itr_insert.first->second.emplace(id);
   }
+
   // Loop over modules
   edm::LogPrint("TreeFilling") << "Filling modules tree";
+
+  std::cout << " looping in modules " << std::endl;
```

Comment:
This `cout` doesn't bring much information

Reply:
done